libmscore/rest.cpp

```diff
@@ -1025,7 +1074,7 @@ Shape Rest::shape() const
                   int n    = measure()->mmRestCount();
                   std::vector<SymId>&& s = toTimeSigString(QString("%1").arg(n));
                   qreal x  = _mmWidth * .5;
-                  qreal y  = -_spatium * 1.5 - staff()->height() *.5;
+                  qreal y = _yPosMMRestNumber * spatium() - staff()->height() *.5;
```

Comment:
I know it was already there before, but there's a missing space between the last * and `.5`. It should be `* .5` instead of `*.5`.

Btw, I've seen quite a few * .5 in the last days. They are not rare in the codebase. Is there a reason not to use `/ 2` instead?

Reply:
I think maybe that would cause an implicit conversion from `int` to `double` (or, as we're calling it here, `qreal`). It could, of course, be `/ 2.0`, but that requires typing two extra characters!

Reply:
Some would claim floating point multiplication is faster, which is true in general.